### PR TITLE
fix(rollback): Should be passing target groups and security groups in request

### PR DESCRIPTION
**Pull request by jeyrschabu**
- include request target groups and security groups

### CODE REVIEW

**Comment by jeyrschabu** [comment]
Not passing target groups through causes the rollback to fail in titus. This fixes that.
`Deck` should ensure the parameters are passed in every request.

**Comment by ajordens** [comment]
Didn't actually run this but looks reasonable to me.

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/PreviousImageRollback.groovy [hunk 1]
```diff
@@ -97,6 +97,8 @@ class PreviousImageRollback implements Rollback {
       credentials                  : parentStageContext.credentials,
       cloudProvider                : parentStageContext.cloudProvider,
       delayBeforeDisableSec        : delayBeforeDisableSeconds ?: 0,
+      targetGroups                 : parentStageContext.targetGroups,
```
Comment (marchello2000) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/PreviousImageRollback.groovy]: what if this deploy changes the `targetGroup` or `securityGroup` to something new, shouldn't we restore it to the original state?

Comment (marchello2000) [reply orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/PreviousImageRollback.groovy]: Actually, looks like only `deck` uses `PREVIOUS_IMAGE`. Do you have a corresponding `deck` PR for this as well?

Comment (jeyrschabu) [reply orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/PreviousImageRollback.groovy]: Will fire up a deck PR as a follow up to re-enable Titus previous image rollbacks and ensure target groups are passed through the request.

**Pull request merged**